src/components/schemas/account.yaml

```diff
@@ -0,0 +1,35 @@
+account_out:
+  type: object
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: 'accrigoog2'
+    created_at:
+      $ref: './fields.yaml#/created_at'
+    updated_at:
+      $ref: './fields.yaml#/updated_at'
+    user_id:
+      $ref: './fields.yaml#/fk_string'
+#    name:
+#      type: string
+#      enum:
+#        - Personal Account
+#        - Corporate Credit Card Account
+#        - Advance Account
+    type:
+      type: string
+      enum:
+        - PERSONAL_ACCOUNT
+        - PERSONAL_CORPORATE_CREDIT_CARD_ACCOUNT
+        - PERSONAL_ADVANCE_ACCOUNT
+    currency:
+      $ref: './fields.yaml#/currency'
+    target_balance_amount:
```

Comment:
I don't think this is necessary - can you please remove it? It can be hidden field from sync for now. It has no use for an external client afaict.